### DISCUSSION TITLE
How to launch webui

### DISCUSSION THREAD

**Discussion by Lcs-pb** [lllyasviel/Fooocus#1172] [Q&A]
Hi, I have installed the windows zip, and i've run the run.bat file, afterwards the application opened a tab. However, now that the application stopped running, how do I run it again? I can't seem to run any of the python scripts, is this expected behaviour? Thanks, in advance!

**Comment by alfred-delacosta** [comment]
To run it again, just run the run.bat file. That will start the server. As long as you keep the CMD window open, it will keep the server running.
